src/node/api_clients/base_client.py

```diff
@@ -1,30 +1,30 @@
-import logging
 import requests
 from tenacity import retry, stop_after_delay, wait_fixed
 from abc import ABC, abstractmethod
+from src.libs.custom_logger import get_custom_logger
 
-logger = logging.getLogger(__name__)
+logger = get_custom_logger(__name__)
 
 
 class BaseClient(ABC):
     # The retry decorator is applied to handle transient errors gracefully. This is particularly
     # useful when running tests in parallel, where occasional network-related errors such as
     # connection drops, timeouts, or temporary unavailability of a service can occur. Retrying
     # ensures that such intermittent issues don't cause the tests to fail outright.
-    @retry(stop=stop_after_delay(2), wait=wait_fixed(0.1), reraise=True)
+    @retry(stop=stop_after_delay(0.5), wait=wait_fixed(0.1), reraise=True)
     def make_request(self, method, url, headers=None, data=None):
         logger.debug("%s call: %s with payload: %s", method.upper(), url, data)
         response = requests.request(method.upper(), url, headers=headers, data=data)
         try:
             response.raise_for_status()
         except requests.HTTPError as http_err:
-            logger.error("HTTP error occurred: %s", http_err)
+            logger.error("HTTP error occurred: %s. Response content: %s", http_err, response.content)
```

Comment:
lazy logging since to be the better approach when comes to logging vs compared to f-strings
There are pros and cons and different opinions for both if you google it (ex https://medium.com/flowe-ita/logging-should-be-lazy-bc6ac9816906)
I've used both ways in the past, I don't have a personal preference. Most important thing is to have them consistent.

Reply:
I've updated all logs to use f-strings to be consistent with vars, Exceptions etc that use f-strings

Reply:
Oh, I didn't know! That's actually rather interesting! It hadn't occurred to me that the `*args` of the logging method would be chained that way! I saw the percentages and I assumed it was the typical (old) string format; good to know!

I take my comment back, both approaches have their advantages!